modAionImpl/src/org/aion/zero/impl/config/CfgAion.java

```diff
@@ -136,6 +139,51 @@ private void closeFileInputStream(final FileInputStream fis) {
         }
     }
 
+    public void setForkProperties() {
+        setForkProperties("mainnet");
+    }
+
+    public void setForkProperties(String networkName) {
+        Properties properties = new Properties();
+
+        // old kernel doesn't support the fork feature.
+        if (networkName.equals("config")) {
```

Comment:
Why is this needed?  Maybe I am missing something, but if this code is executing, doesn't  that mean they're using the new kernel? 

Or is this just to support the old kernel's config format?

Reply:
just for someone still uses old config struct to execute kernel.

Reply:
Ok.  PR overall looks good to me.  Maybe consider putting a warning message for that case.

Reply:
the reason doesn't print string here cause it will cause the packaging process failed.